sites/fast-website/src/app/css/style.css

```diff
@@ -227,10 +227,9 @@ fast-badge::part(control) {
   line-height: var(--type-ramp-plus-1-line-height);
 }
 
-.divider {
+.footer-divider {
   grid-column: 2;
-  margin-top: 100px;
-  margin-bottom: 60px;
+  margin-top: 300px;
```

Comment:
This seems excessive

Reply:
@radium-v in figma they had the space between the cards in community and the divider in the footer being about 340px. I was trying to replicate that with this margin(I already decreased it to 300px). I can reduce it further down to 200px or 100px if you would like?